js/Items.js

```diff
@@ -0,0 +1,99 @@
+angular.module('PathOfDamage')
```

Comment:
I feel like I go through a lot of levels of indentation just to return a single array, but I didn't find a cleaner way to do this.